### PR TITLE
[NewUI] SendV2-#3: 'To' input component

### PR DESCRIPTION
**Pull request by danjm**
Implements the 'To' component for send-v2

@chikeichan I think we need to style the options on the dropdown.

screen shot 2017-10-06 at 7 47 15 am
screen shot 2017-10-06 at 7 48 54 am

### CODE REVIEW

File: ui/app/components/identicon.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ IdenticonComponent.prototype.componentDidMount = function () {
   if (!address) return
 
   var container = findDOMNode(this)
-
+  console.log(`address`, address);
```
Comment (sdtsui) [inline ui/app/components/identicon.js]: stray log

File: ui/app/components/send/from-dropdown.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+inherits(FromDropdown, Component)
+function FromDropdown () {
+  Component.call(this)
+}
+
+FromDropdown.prototype.renderSingleIdentity = function (
+  account, 
+  handleClick, 
+  inList = false,
```
Comment (sdtsui) [inline ui/app/components/send/from-dropdown.js]: I'd consider handling `inList` differently. This is a function with a boolean variable for control flow...when you use it anywhere else, it adds another layer of abstraction for the reader to figure out.

See L66 for an example, it's hard to tell what `true` means.  What do you thinkn of moving that check out (whenever it gets implemented) into a function?

**Review by sdtsui — APPROVED** [review]
@danjm - LGTM, added a minor comment on a func sig I thought could be improved. 
/cc @chikeichan

**Comment by sdtsui** [comment]
@danjm  +1 on styling the `to` component options.

**Comment by danjm** [comment]
@chikeichan resolved conflicts here

**Pull request merged**